### DISCUSSION TITLE
This application failed to start because no Qt platform plugin could be initialized

### DISCUSSION THREAD

**Discussion by pratikshirke0906** [roboflow/supervision#933] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

please hekp me sove this

### Additional

(venv2) pratik@DESKTOP-8QUD4SD:~/supervision/examples/speed_estimation$   python yolo_nas_example.py \
  --source_video_path data/vehicles.mp4 \
  --target_video_path data/vehicles-result.mp4 \
  --confidence_threshold 0.3 \
  --iou_threshold 0.5
The console stream is logged into /home/pratik/sg_logs/console.log
[2024-02-17 16:42:16] INFO - crash_tips_setup.py - Crash tips is enabled. You can set your environment variable to CRASH_HANDLER=FALSE to disable it
[2024-02-17 16:42:16] WARNING - __init__.py - Failed to import pytorch_quantization
[2024-02-17 16:42:19] WARNING - calibrator.py - Failed to import pytorch_quantization
[2024-02-17 16:42:19] WARNING - export.py - Failed to import pytorch_quantization
[2024-02-17 16:42:19] WARNING - selective_quantization_utils.py - Failed to import pytorch_quantization
[2024-02-17 16:42:19] INFO - checkpoint_utils.py - License Notification: YOLO-NAS pre-trained weights are subjected to the specific license terms and conditions detailed in
https://github.com/Deci-AI/super-gradients/blob/master/LICENSE.YOLONAS.md
By downloading the pre-trained weight files you agree to comply with these terms.
[2024-02-17 16:42:23] INFO - checkpoint_utils.py - Successfully loaded pretrained weights for architecture yolo_nas_l
[2024-02-17 16:42:23] INFO - pipelines.py - Fusing some of the model's layers. If this takes too much memory, you can deactivate it by setting `fuse_model=False`
qt.qpa.xcb: could not connect to display
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/pratik/supervision/examples/speed_estimation/venv2/lib/python3.10/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb.

Aborted

**Comment by SkalskiP** [comment]
Hi @pratikshirke0906 👋🏻 

Let's start by converting this issue into a discussion and placing it in the Q&A section.

**Comment by SkalskiP** [comment]
@pratikshirke0906, what operating system are you using?

**Comment by onuralpszr** [comment]
@pratikshirke0906  could you make sure install python-opencv instead of python-opencv-headless
